packages/react-examples/package.json

```diff
@@ -58,7 +58,8 @@
     "@fluentui/react-label": "^9.0.0-alpha.7",
     "@fluentui/react-link": "^9.0.0-alpha.45",
     "@fluentui/react-make-styles": "^9.0.0-alpha.41",
-    "@fluentui/react-menu": "^9.0.0-alpha.33",
+    "@fluentui/react-menu": "^9.0.0-alpha.35",
```

Comment:
hmm is this a bug in the migration ?  both of these should have been removed by migration

Reply:
oh outdated ? I guess I checked the PR while you've been rebasing ? :D